x-pack/plugins/security_solution/common/api/detection_engine/rule_management/export_rules/export_rules_route.schema.yaml

```diff
@@ -51,7 +51,7 @@ paths:
         200:
           description: Indicates a successful call.
           content:
-            application/ndjson:
+            application/json:
```

Comment:
If this is `ndjson`, the autogeneration doesn't produce a type for it. So we can either have it like this, or do some hacking in the autogeneration so that for the Export endpoint no response object is attempted to be imported and used to type the request method.